lib/github_changelog_generator/parser.rb

```diff
@@ -6,36 +6,23 @@
 require "github_changelog_generator/helper"
 
 module GitHubChangelogGenerator
-  class Parser
-    # parse options with optparse
-    def self.parse_options
-      options = default_options
-
-      ParserFile.new(options).parse!
+  class ArgvParser
```

Comment:
In order to make the file layout more regular, could this class be in its own file?

Reply:
It should be yes. I'll adjust